### PR TITLE
Load deserializer early. Fixes #115.

### LINKED ISSUE #115: PDF View does not reopen
Open a pdf. Quit atom. Restart atom. PDF View is not reloaded.

### PR DESCRIPTION
**Pull request by 314eter**
This will need a small update to get it working with [the new package.json format](https://github.com/atom/atom/pull/10764), but because that will break compatibility with atom's stable branch, I suppose we'll have to wait?

### CODE REVIEW

**Comment by 314eter** [comment]
Works for latest stable and master now.

File: lib/pdf-editor.js [hunk 1]
```diff
@@ -28,12 +13,15 @@ export function deactivate() {
...
-const pdfExtensions = new Set(['.pdf']);
+let pdfExtensions = null;
 
 function openUri(uriToOpen) {
   if (path === null) {
     path = require('path');
   }
+  if (pdfExtensions === null) {
+    pdfExtensions = new Set(['.pdf']);
+  }
```
Comment (izuzak) [inline lib/pdf-editor.js]: What's the benefit of this change vs the old behavior?

File: package.json [hunk 2]
```diff
@@ -15,5 +18,20 @@
...
     "underscore-plus": "^1.6",
     "atom-space-pen-views": "^2.0.3"
+  },
+  "configSchema": {
+    "reverseSyncBehaviour": {
+      "type": "string",
+      "enum": ["Disabled", "Click", "Double click"],
+      "default": "Click",
+      "title": "SyncTeX Reverse sync behaviour",
+      "description": "Specify the action on the PDF generated with the `--synctex=1` option that takes you to the source."
```
Comment (izuzak) [inline package.json]: The above changes seem unrelated to what this pull request is fixing. This pull request should be only about the deserializer, not about changing config options so that double-click is supported for synctex and not just click.

**Comment by izuzak** [comment]
Seems like there are more problems in Atom land related to this https://github.com/atom/atom/issues/10995. I'm going to hold of on merging this until that's resolved. Also, it seems that this pull request contains some unrelated changes. Can we keep this pull request focused on fixing #115 and leave other changes out?

**Comment by 314eter** [comment]
I committed some unrelated changes by accident. They are reverted now.

**Comment by izuzak** [comment]
Sweet -- thanks for sticking with this, @314eter :zap:

**Comment by izuzak** [comment]
@314eter Looking forward to the followup PR which you mentioned here: https://github.com/izuzak/atom-pdf-view/pull/116#issuecomment-184020894

Also, I think I just noticed another problem. If I open several PDF files, then close Atom, then open Atom again -- all previously opened PDF files will be listed in tabs, but only the PDF file which was last focused before closing Atom will be correctly loaded while other PDF files will be blank. Wondering if you have an idea about why that's so. If not, I'll just open an issue to track this problem. :v:

**Comment by 314eter** [comment]
Do you have the fitToWidthOnOpen option set? The clientWidth of unfocused tabs is 0, so the scale is set to 0. This should be fixed by #122.
